src/cli/cli_tcat.cpp

```diff
@@ -200,6 +216,39 @@ template <> otError Tcat::Process<Cmd("vendorid")>(Arg aArgs[])
     return error;
 }
 
+template <> otError Tcat::Process<Cmd("devid")>(Arg aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    if (aArgs[0].IsEmpty())
+    {
+        OutputLine("TCAT GeneralId:");
+        if (sGeneralDeviceId.mDeviceIdLen != 0)
+        {
+            OutputBytesLine(sGeneralDeviceId.mDeviceId, sGeneralDeviceId.mDeviceIdLen);
+        }
+        else
+        {
+            OutputLine("empty");
+        }
+        ExitNow();
+    }
+
+    if (aArgs[0] == "empty")
```

Comment:
(see previous comments on 'clear':) actually here we could easily change it to clear without having to touch the device type ID "empty".  So maybe change this to "clear" ?

Reply:
Changed to clear